src/vivarium_gates_bep/components/artifact_builder.py

```diff
@@ -204,6 +206,44 @@ def write_iron_deficiency_data(artifact, location):
         write(artifact, key, load(key))
 
 
+def get_lbwsg_for_loc(key, location, default_loader):
+    mtype, category, measure = key.split('.')
+    if location in bep_globals.LBWSG_MAPPER:
+        sanitized_location =  location.replace(" ", "_").replace("'", "-")
+        store = pd.HDFStore(f'{bep_globals.LBWSG_PATH}/{sanitized_location}.hdf', 'r')
```

Comment:
I use store all the time as a habit, but I think you should use Artifact here. Esp. because the keys are actually dotted, which is what our tooling uses, not slash demarcated

Reply:
Changed.